packages/ssx-server/src/middlewares/express/endpoints.ts

```diff
@@ -1,9 +1,9 @@
 import express from 'express';
 import { Request, Response } from 'express';
 import { SSXServer } from '../../server';
-import { SSXServerRoutes } from '@spruceid/ssx-core';
+import { SSXServerRouteNames } from '@spruceid/ssx-core';
 
-const ssxEndpoints = (ssx: SSXServer, routes?: SSXServerRoutes) => {
+const ssxEndpoints = (ssx: SSXServer, routes?: SSXServerRouteNames) => {
```

Comment:
This function is configuring the route path, but not the method. On lines 18, 36, and 96 the methods are hard coded.

Reply:
This was intentional! For our package, since we are providing the endpoints and their implementation, allowing the names of the endpoints to be configured should be good enough. There is not much benefit to allowing for configuring the endpoint method if they are using our library, especially if the cost is having to make the endpoints change (get vs post vs put, etc.)

Reply:
Are we ok with letting the user change this on the client but receive an error on the server (middleware)?

Reply:
Yes! If a user is changing this endpoint (on the client or server) it's should be because they have a specific reason not to use the existing enpoints. We can mention this in the docs that if they need to change the endpoints in both places if they are using ssx in both the client and the server.

Reply:
ok!